src/lib/abis/v2/index.ts

```diff
@@ -1,5 +1,30 @@
+import { BackersManagerAbi } from './BackersManagerAbi'
+import { BuilderRegistryAbi } from './BuilderRegistryAbi'
+import { CycleTimeKeeperAbi } from './CycleTimeKeeperAbi'
+import { GaugeAbi } from './GaugeAbi'
+import { RewardDistributorAbi } from './RewardDistributorAbi'
+
 export * from './BackersManagerAbi'
 export * from './BuilderRegistryAbi'
 export * from './CycleTimeKeeperAbi'
 export * from './GaugeAbi'
 export * from './RewardDistributorAbi'
+
+export type BackersManagerAbi = typeof BackersManagerAbi
+export type BuilderRegistryAbi = typeof BuilderRegistryAbi
+export type CycleTimeKeeperAbi = typeof CycleTimeKeeperAbi
+export type GaugeAbi = typeof GaugeAbi
+export type RewardDistributorAbi = typeof RewardDistributorAbi
+
+export const abis = {
+  BackersManagerAbi,
+  BuilderRegistryAbi,
+  CycleTimeKeeperAbi,
+  GaugeAbi,
+  RewardDistributorAbi,
+} as const
+export type CollectiveRewardsAbis = typeof abis
+export type CollectiveRewardsAbiName = keyof CollectiveRewardsAbis
+export type CollectiveRewardsAbi = CollectiveRewardsAbis[CollectiveRewardsAbiName]
+
+export const getAbi = (abiName: CollectiveRewardsAbiName): CollectiveRewardsAbi => abis[abiName]
```

Comment:
What's the purpose of this function if we export both the ABIs directly and the `abis` object?

Reply:
1. single point of reference
2. no need to use the badly formed abi var namig that may be confused with type
3. easy to use without having to remember the exact name
4. prevents typos 
5. easier to test (can mock the getAbi function to return a fake abi)